openmdao/recorders/tests/test_sqlite_recorder.py

```diff
@@ -643,7 +642,15 @@ def test_record_line_search_armijo_goldstein(self):
         coordinate = [0, 'Driver', (0,), 'root._solve_nonlinear', (0,), 'NewtonSolver', (3,), 'ArmijoGoldsteinLS', (4,)]
         expected_abs_error = 3.49773898733e-9
         expected_rel_error = expected_abs_error / 2.9086436370499857e-08
-        expected_solver_output = None
+        expected_solver_output = {
+            "con_cmp1.con1": [-22.42830237000701],
```

Comment:
that's a lot of precision in these tests, the numbers don't vary at all across platforms?

Reply:
Oh I'm sure the numbers do vary across platforms, though the assertions on these tests check that the value is in an expected range defined by the tolerance `self.eps`

Reply:
ah right, of course...  just seemed a little odd to see all those digits :)